moveit_ros/moveit_servo/include/moveit_servo/pose_tracking.h

```diff
@@ -81,11 +82,12 @@ const std::unordered_map<PoseTrackingStatusCode, std::string> POSE_TRACKING_STAT
  * Class PoseTracking - subscribe to a target pose.
  * Servo toward the target pose.
  */
-class PoseTracking
+class PoseTracking 
```

Comment:
Remove space at the end of this line.

https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut